examples/tls-inspector/verify.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -e
+
+export NAME=tls-inspector
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+run_log "Test tls inspector"
+responds_without \
+    '"x-forwarded-proto"' \
+    -k \
+    https://localhost:10000
+curl -sk --http1.1 https://localhost:10000  | jq  '.os.hostname'
+curl -sk --http2 https://localhost:10000  | jq  '.os.hostname'
+curl -sk http://localhost:10000  | jq  '.os.hostname'
+
+curl -sk http://localhost:12345/stats |grep tls_inspector
```

Comment:
also with this one we might want to add

```
| grep "tls_inspector.alpn_found: 3"
```
and perhaps add a couple more for sni/tls/tls_not_found to test that the admin stats are as expected

Reply:
I would like show all stats of tls inspector. What about getting stats after each curl, then user could see the stats change.

Reply:
this is for the test - the user wouldnt normally run this

Reply:
Got it.